### PR TITLE
improvements to harmonize function

### PR DESCRIPTION
**Pull request by knaaptime**

### CODE REVIEW

**Comment by renanxcortes** [comment]
Does this fix the mismatching arrays issue?

**Comment by knaaptime** [comment]
no, that needs to be done in tobler

**Comment by renanxcortes** [comment]
> no, that needs to be done in tobler

Yes, yes, I'll open an issue for that!

File: geosnap/harmonize/harmonize.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import pandas as pd
 import geopandas as gpd
+from warnings import warn
```
Comment (sjsrey) [inline geosnap/harmonize/harmonize.py]: Where is warn being used?

Comment (knaaptime) [reply geosnap/harmonize/harmonize.py]: good catch. i swapped it out later for the ValueError

**Comment by renanxcortes** [comment]
> > no, that needs to be done in tobler
> 
> Yes, yes, I'll open an issue for that!

The issue I refereed here is https://github.com/pysal/tobler/issues/2.

**Pull request merged**